### PR TITLE
Deere: redesign fader and knob colors

### PR DESCRIPTION
**Pull request by Be-ing**
Continuing #1399.
Before:
screenshot from 2018-04-01 21-11-51

After:
deere-knob-colors

### CODE REVIEW

**Comment by ronso0** [comment]
The purple and green rings are a bit too dark, in bright light they are almost invisible and with normal lighting they appear much thinner than the other rings.
What about making them a bit brighter, shifting the hue and lightness more towards magenta and lime (not the neon tones though)?

**Comment by Be-ing** [comment]
Good idea. I was working on this at night. Now I brought my laptop into sunlight to test it. How is this?
screenshot from 2018-04-02 07-13-21

**Comment by daschuer** [comment]
Can you summarize, what you are trying to fix with this PR? The "Before" version looks quite elegant in my eyes.

**Comment by Be-ing** [comment]
IMO the old orange and gray colors feel dull. My goals here:

  - Make volume faders easy to see from across a room
  - Feel lively and colorful but not annoying/in-your-face

**Comment by daschuer** [comment]
IMHO it is impossible to optimize a dark skin for bright sunlight, since it requires justed the opposite. Can't we hack up a high   contrast color scheme for Deere optimized for daylight events.

**Comment by Be-ing** [comment]
This looks pretty decent in sunlight to me, but that's not the main purpose of this PR anyway.

**Comment by Be-ing** [comment]
We've come a long way from 2.0!
screenshot from 2018-04-02 08-30-07

**Comment by daschuer** [comment]
> This looks pretty decent in sunlight to me, but that's not the main purpose of this PR anyway.

Dark themes are in general not a good choice for outdoor applications. The brightness off the theme schould match the surrounding conditions, to avoid that the eyes need to adopt when moving from the controller to the screen. This is especially true when you use glare screens, which becomes a mirror with dark backgrounds used outdoor.

I have just found this palette: http://ethanschoonover.com/solarized can we use it for Deere?

**Comment by Be-ing** [comment]
> I have just found this palette: http://ethanschoonover.com/solarized can we use it for Deere?

Now is not the time to redesign the entire skin. I am only doing finishing touches with this PR.

That color palette looks interesting though. I may use it in the 4x1 decks skin I'll work on after 2.1.

**Comment by Be-ing** [comment]
Ready for merge?

**Comment by Be-ing** [comment]
@ronso0 any more comments? I would like to finish this to take the screenshots for #1525.

**Comment by ronso0** [comment]
Actually I don't see an advantage in changing the EQ & Gain colors, and the fader colors are quite dominant now. The color choice is definitely a matter of taste and IMO we should test visibility for all skins  on different screens under different lighting conditions for the next (point) release.
But for now this LGTM

**Comment by Be-ing** [comment]
Okay thank you for review.
